src/coreclr/pal/src/misc/cgroup.cpp

```diff
@@ -40,6 +40,8 @@ SET_DEFAULT_DEBUG_CHANNEL(MISC);
 #define CGROUP_MEMORY_STAT_FILENAME "/memory.stat"
 #define CGROUP1_MEMORY_USAGE_FILENAME "/memory.usage_in_bytes"
 #define CGROUP2_MEMORY_USAGE_FILENAME "/memory.current"
+#define CGROUP1_MEMORY_USE_HIERARCHY_FILENAME "/memory.use_hierarchy"
```

Comment:
Is the cgroup memory limit implementation in the Win32 PAL used for anything?

Reply:
That's a good point, I haven't realized we've stopped using that once we've moved embedded GC to its own PAL.